landsat/uploader.py

```diff
@@ -11,16 +11,20 @@
 import time
 import threading
 import contextlib
-import Queue
+try:
+    import queue
+except ImportError:
+    import Queue as queue
 from multiprocessing import pool
 try:
     import cStringIO
     StringIO = cStringIO
 except ImportError:
-    import StringIO
+    from io import StringIO
+    #import StringIO        # FIXME: why original fall back to StringIO in Py2?
```

Comment:
The original use StringIO as a fall back if cStringIO failed to import. When does this happen, and is it a concern?
Also, need to look carefully at the return types were io.StringIO is used.